### DISCUSSION TITLE
Help with SPIFF Images.....

### DISCUSSION THREAD

**Discussion by bcodermicro** [ImpulseAdventure/GUIslice#318] [General]
So i've attempted to do the following steps but end up with a GREY square on the display where the iimage is supposed to be....

In the builder I add an image.....
I've added 12 images to the SPIFF area of my ESP32. 
I can print out the file list...
`20:51:46.945 -> File: /partnight_sm.jpg
20:51:46.945 -> File: /downpour_sm.jpg
20:51:46.945 -> File: /thunderstorm_sm.jpg
20:51:46.945 -> File: /clearnight_sm.jpg
20:51:46.945 -> File: /snow_sm.jpg
20:51:46.945 -> File: /snowshowers_sm.jpg
20:51:46.945 -> File: /Clouds_sm.jpg
20:51:46.945 -> File: /partsun_sm.jpg
20:51:46.945 -> File: /dangcold_sm.jpg
20:51:46.945 -> File: /verycloud_sm.jpg
20:51:46.945 -> File: /thunder_sm.jpg
20:51:46.981 -> File: /windy_sm.jpg
20:51:46.981 -> File: /Sunny_sm.jpg
20:51:46.981 -> File: /danghot_sm.jpg
`

I set the image to one in the file list..... uaing the builder....

`#define IMG_CLEARNIGHT_SM         "/clearnight_sm.jpg"` 

` pElemRef = gslc_ElemCreateImg(&m_gui,E_ELEM_IMAGE1,E_PG_MAIN,(gslc_tsRect){215,17,70,70},
    gslc_GetImageFromFile(IMG_CLEARNIGHT_SM,GSLC_IMGREF_FMT_JPG));
`

the above appears in the _GSLC.h file. 

I also made sure that in the GUIslice config file I changed 
  #define GSLC_SPIFFS_EN               1 
 in the 

esp-tftespi-default-notouch.h

Still end up with the grey square thoug.....am I missing  some stepss...

**Comment by bcodermicro** [comment]
Solved in Isssues...

**Reply by ImpulseAdventure** [reply]
Thanks... I had opened up the discussions but missed the first post :)
For future reference for others, the issue that resolved this is can be found in #319
